lib/utilities/legacyAWSBehavior.js

```diff
@@ -0,0 +1,24 @@
+import config from '../Config';
+
+/**
+ * check the legacy AWS behaviour
+ * For new configuration: check if locationConstraint has its
+ * legacyAwsBehavior put to true
+ * For old configuration: check if locationConstraint is 'us-east-1' and
+ * usEastBehavior is true
+ * @param {string | undefined} locationConstraint - value of user
+ * metadata location constraint header or bucket location constraint
+ * @param {boolean} usEastBehavior - only used for old configuration
+ * @return {boolean} - true if valid, false if not
+ */
+
+export default function isLegacyAwsBehavior(locationConstraint,
+  usEastBehavior) {
```

Comment:
I am going to keep it like it for now because createBucket https://github.com/scality/S3/blob/master/lib/api/apiUtils/bucket/bucketCreation.js#L161
has been tested (unit tests) https://github.com/scality/S3/blob/master/tests/unit/bucket/bucketCreation.js#L20 with `usEastBehavior` true and false.